src/task.ts

```diff
@@ -59,7 +59,7 @@ export type Task<A extends string = never> = {
   do: Location | (() => void);
   post?: () => void;
 
-  acquire?: AcquireItem[];
+  acquire?: (AcquireItem | (() => AcquireItem))[];
```

Comment:
Maybe `AcquireItem[] | () => AcquireItem[]` instead? To make it easier to dynamically gather different lists of items